tests/unit/SubscriptionUtilsTest.ts

```diff
@@ -14,7 +14,7 @@ const billingGraceEndPeriod: BillingGraceEndPeriod = {
 };
 
 const GRACE_PERIOD_DATE = new Date().getTime() + 1000;
-const GRACE_PERIOD_DATE_OVERDUE = new Date().getTime() - 1000;
+const GRACE_PERIOD_DATE_OVERDUE = new Date().getTime() - 1000*3600;
```

Comment:
Why are we updating `GRACE_PERIOD_DATE_OVERDUE`?

Reply:
Fixed!